docs/KNOWN_ISSUES.md

```diff
@@ -49,3 +63,31 @@ If you *must* call `player.reset()` then you'll have to manually restart the `vi
         // Add your plugin configuration here
     });
     ```
+
+## Setting start position on iOS safari
+
+On iOS Safari, there is a compatibility issue with the `loadedmetadata` event when attempting to set the start position
+of a video. This event, commonly used for this purpose on other browsers, may lead to unexpected behavior on iOS
+devices.
+
+### Issue Details
+
+- **Scenario:** Calling `player.on('loadedmetadata', () => player.currentTime(startPosition))`.
```

Comment:
Since the function is called `currentTime` why not `startTime`? This will ensure consistency with the function name.

Reply:
No problem, I've changed that.